tests/unittest/test_breakpad_resource.py

```diff
@@ -19,6 +19,34 @@
 from testlib.mini_poster import compress, multipart_encode
 
 
+class AnyTagValue:
```

Comment:
In the other PR, I did a ridiculous thing to solve the same problem this solves because I thought I couldn't solve it this way on account of how Markus `filter_records` works. But then I decided to try it out and it works super. I'll fix Markus and then we can remove this and the ridiculous thing I did in that other PR.

The thing this does is two fold:

1. `AnyTagValue("host") == "host:somevalue"`
2. `AnyTagValue("host")` will sort correctly with other strings; for example these will sort the same:
   ```
   foo = ["host:12345", "environment:stage"]
   foo = [AnyTagValue("host"), "environment:stage"]
   ```